internal/app/handler/handler.go

```diff
@@ -88,10 +91,41 @@ func (h *Handler) Handle(ctx context.Context, event sdk.Event) error {
 
 func NewEnv(cr *opv1.App) ([]runtime.Object, error) {
 	var objs []runtime.Object
-	env, err := defaults.GetEnvironment(cr)
+	env, pw, err := defaults.GetEnvironment(cr)
```

Comment:
Please use "password", or if you can't, passwd or similar that is more clear and less ambiguous.